lib/ansible/modules/packaging/language/pip.py

```diff
@@ -235,6 +241,73 @@
                              'pip': 'import pkg_resources; print(pkg_resources.get_distribution("pip").version)'}
 
 
+class Distribution:
```

Comment:
Could you, please, write PEP257-compatible docstrings to all public interfaces you add?